src/print.rs

```diff
@@ -95,6 +95,14 @@ impl<'a, 'b> BacktraceFmt<'a, 'b> {
     pub fn message(&mut self, msg: &str) -> fmt::Result {
         self.fmt.write_str(msg)
     }
+
+    /// Return the inner formatter.
+    ///
+    /// This is used for writing custom information between frames with `write!` and `writeln!`,
+    /// and won't increment the `frame_index` unlike the `frame` method.
+    pub fn fmt(&mut self) -> &mut fmt::Formatter<'b> {
```

Comment:
With this we're opening up expressions like `displayer.fmt(bt.fmt())` and `bt.fmt().write_fmt(format_args!("{} foo {:?}", 1, 2))`

Here, everything is local and it's clear what the field is, but to head off semantic satiation and/or confusion when it is non-local, could we name this function anything else? `fmtter`?

Reply:
yes, `fmtter` is a better name.
or `get_fmt`?

Reply:
Or `formatter` or whatever seems good.

Reply:
Fixed, thanks!